pandas/tests/io/test_parquet.py

```diff
@@ -410,11 +410,25 @@ def test_write_multiindex(self, pa):
         check_round_trip(df, engine)
 
     def test_write_column_multiindex(self, engine):
-        # column multi-index
+        # Not able to write column multi-indexes with non-string column names.
         mi_columns = pd.MultiIndex.from_tuples([("a", 1), ("a", 2), ("b", 1)])
         df = pd.DataFrame(np.random.randn(4, 3), columns=mi_columns)
         self.check_error_on_write(df, engine, ValueError)
 
+    def test_write_column_multiindex_string(self, pa):
+        # GH #34777
+        # Not supported in fastparquet as of 0.1.3 or older pyarrow version
```

Comment:
are we > than the min pyarrow version?

Reply:
Based on the min versions listed in pandas dependencies, the min pyarrow version is 0.15 while we are currently at 0.16 - at least for the dev environment that I'm working on.